### PR TITLE
Lesson 12

### PR DESCRIPTION
**Pull request by JohnWickkk**

### CODE REVIEW

File: Lesson_12 (OOP begining + func)/HomeWork_12_1.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+if __name__ == "__main__":
+
+    laptop_brand = input('Enter your laptop brand (for axample, Toshiba): ')
+
+    if laptop_brand == 'Toshiba':
+        print(get_upgrade())
+        print(f"Investment return for 1 Toshiba share is {get_investment()} uah")
+    else:
+        print(f"We don't have information about {laptop_brand} upgrades or investments.")
```
Comment (MarinaZelenska) [inline Lesson_12 (OOP begining + func)/HomeWork_12_1.py:22]: Our topic is OOP. This is not a OOP way .

File: Lesson_12 (OOP begining + func)/HomeWork_12_2.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+"""
+Implement your own implementation of function max__
+Implement your own min function__
+"""
+
+numbers = []
+
+
+def _min_max(numbers):
```
Comment (MarinaZelenska) [inline Lesson_12 (OOP begining + func)/HomeWork_12_2.py:9]: This task should show your practice with *args. But you don't use it

Comment (MarinaZelenska) [reply Lesson_12 (OOP begining + func)/HomeWork_12_2.py:9]: Function shouldn't start with _

**Comment by JohnWickkk** [comment]
переробити

File: Lesson_12 (OOP begining + func)/HomeWork_12_2.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    minimum = numbers[0]
+    maximum = numbers[0]
+
+    for num in numbers[1:]:
+        if num < minimum:
+            minimum = num
+        elif num > maximum:
+            maximum = num
+
+    return minimum, maximum
```
Comment (MarinaZelenska) [inline Lesson_12 (OOP begining + func)/HomeWork_12_2.py:53]: image

**Pull request closed without merging**